doc/release-notes.md

```diff
@@ -102,8 +102,15 @@ will trigger BIP 125 (replace-by-fee) opt-in. (#11413)
   option `-deprecatedrpc=banscore` is used. The `banscore` field will be fully
   removed in the next major release. (#19469)
 
-- The `testmempoolaccept` RPC returns `vsize` and a `fee` object with the `base` fee
-  if the transaction passes validation. (#19940)
+- User-configured maximum feerates are no longer considered network rules. The
```

Comment:
The never have been? The code happened to be in ATMP, but never executed for network txs.
```suggestion
- The
```

Reply:
I guess I was trying to justify the error code change but yeah it's irrelevant here.